GVFS/GVFS.Virtualization/Projection/GitIndexProjection.FolderData.cs

```diff
@@ -49,6 +51,26 @@ public void Include()
                 }
             }
 
+            public string HashedFileShas()

```

Comment:
Rename this method since it will be file shas and folders that will be hashed.

Reply:
perhaps `HashedChildrenNamesSha()`